doc/Makefile

```
@@ -45,17 +35,15 @@ api:
 	@echo
 	$(SPHINXAUTOGEN) -i -t _templates -o api/generated api/*.rst
 
-
-linkcheck:
-	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/linkcheck
-	@echo
-	@echo "Link check complete; look for any errors in the above output " \
-	      "or in $(BUILDDIR)/linkcheck/output.txt."
-
-doctest:
-	$(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/doctest
-	@echo "Testing of doctests in the sources finished, look at the " \
-	      "results in $(BUILDDIR)/doctest/output.txt."
-
-serve:
+server:
 	cd $(BUILDDIR)/html && python -m http.server 8009
+
+clean:
+	rm -rf $(BUILDDIR)/doctrees
+	rm -rf $(BUILDDIR)/html
```

I think it's OK to just remove the whole `$(BUILDDIR)` directory.
```suggestion
	rm -rf $(BUILDDIR)
```